R-packages/evalcast/R/get_target_response.R

```diff
@@ -52,7 +11,7 @@ get_target_response <- function(signals,
     enframe(name = NULL, value = "forecast_date") %>%
     mutate(incidence_period = incidence_period, ahead = ahead) %>%
     pmap_dfr(get_target_period)
-
+  
```

Comment:
Remove trailing spaces here and in 1 other place below.

Reply:
I think I got these, but maybe check to be sure.

Reply:
LGTM